docs/pages/cloud/faq.mdx

```diff
@@ -42,13 +42,57 @@ Currently there is no way to provide your own bucket.
 
 See our documentation on [data retention](./architecture.mdx#data-retention).
 
-## How do I add nodes to Teleport Cloud?
+## How do I add Nodes to Teleport Cloud?
 
-You can connect servers, kubernetes clusters, databases and applications
+You can connect servers, Kubernetes clusters, databases and applications
 using [reverse tunnels](../setup/admin/adding-nodes.mdx).
 
 There is no need to open any ports on your infrastructure for inbound traffic.
 
+## Which Proxy Service ports are open on my Teleport Cloud tenant?
+
+Teleport Cloud allocates a different set of ports to each tenant. To see which
+ports are available for your Teleport Cloud tenant, run a command similar to the
+following, replacing `mytenant.teleport.sh` with your tenant domain:
+
+```code
+$ curl https://mytenant.teleport.sh/webapi/ping | jq '.proxy'
+```
+
+The output should resemble the following, including the unique ports assigned to
+your tenant:
+
+```json
+{
+  "kube": {
+    "enabled": true,
+    "public_addr": "mytenant.teleport.sh:11107",
+    "listen_addr": "0.0.0.0:3026"
+  },
+  "ssh": {
+    "listen_addr": "[::]:3023",
+    "tunnel_listen_addr": "0.0.0.0:3024",
+    "public_addr": "mytenant.teleport.sh:443",
+    "ssh_public_addr": "mytenant.teleport.sh:11105",
+    "ssh_tunnel_public_addr": "mytenant.teleport.sh:11106"
+  },
+  "db": {
+    "postgres_public_addr": "mytenant.teleport.sh:11109",
+    "mysql_listen_addr": "0.0.0.0:3036",
+    "mysql_public_addr": "mytenant.teleport.sh:11108"
+  },
+  "tls_routing_enabled": true
+}
+```
+
+This output also indicates whether TLS routing is enabled for your tenant. When
```

Comment:
Would it make sense to link "TLS routing" to https://goteleport.com/docs/architecture/tls-routing/ here so users can read about it and understand how it works if they want to? Up to you.